cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -33,6 +33,7 @@ import (
 	bootstrapapi "k8s.io/cluster-bootstrap/token/api"
 	bootstraputil "k8s.io/cluster-bootstrap/token/util"
 	"k8s.io/klog/v2"
+	bootstraptokenv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/bootstraptoken/v1"
```

Comment:
How about group current project imports?
Similar problems exist in other files.

Reply:
Sorry, can you please clarify.
Are you suggesting to change the import alias?

Reply:
ok, i see what you mean.

`k8s.io/kubernetes/cmd/kubeadm/*` should be a separate group under imports.

Reply:
i logged a separate issue for this because ideally we want to fix this in a separate PR:
https://github.com/kubernetes/kubeadm/issues/2515